### ISSUE TITLE
 OT correct masters in sources folder

### ISSUE THREAD

**Issue by dberlow** [TypeNetwork/Amstelvar#118]
The Roman design space file and UFOs In sources folder do not match the build requested.

A. There are trivars at opsz max wght min, e.g. that are not specified for inclusion.
B. There are no 84 pt and 36 pt intermediates on opsz, as specified.

Is there a branch?

**Comment by sannorozco** [comment]
Updated!

Just one question, on A., you mean trivars should be included or not?
Current _Amstelvar-Roman-003.designspace_ has them at the bottom,

Let me know, thanks

**Comment by dberlow** [comment]
There are not supposed to be anything but default, all the axes mins maxs and opsz intermediates for 36 and 84. 

..doing that depended on the previous step of fixing the yopq and etc.
I’m pretty sure doing that was done, and didn’t introduce any other masters and there should be nothing except for the Masters that I just listed.

On Apr 28, 2019, at 11:53 AM, Santiago Orozco <notifications@github.com> wrote:

Updated!

Just one question, on A., you mean trivars should be included or not?
Current Amstelvar-Roman-003.designspace has them at the bottom,

Let me know, thanks
